tests/testthat/test-read_dss_payments.R

```diff
@@ -0,0 +1,27 @@
+# testing read_dss_payments function
+
+test_that("Read of DSS payment Data works", {
+
+  skip_on_ci()      # this function takes a while to run, so better to skip it on ci
+
+  dss <- read_dss_payments()
+
+  expect_s3_class(dss, 'data.frame')
+  expect_equal(class(dss$payments), 'numeric')
+  expect_equal(class(dss$quarter_end_date), 'character')       # probably should be 'Date'?
```

Comment:
for these ones you are correct, please add a mutate to the original functions so that date fields are actually dates.

Reply:
Ok, will do.

Reply:
Do you want me to do this for all the instances where I flagged this, or just for read_dss_payments?

Reply:
please do for all instances where it has been flagged

Reply:
done